PATCH-P0-HACK-UPSTREAM: Fixing readline check

HIST_ENTRY symbol is not defined in main include, but in history.h for
readline. At least in new readline.

Maintainer: Michal Hrusecky <mhrusecky@suse.cz>

Index: config/ac-macros/readline.m4
===================================================================
--- config/ac-macros/readline.m4.orig
+++ config/ac-macros/readline.m4
@@ -4,6 +4,7 @@ AC_DEFUN([MYSQL_CHECK_READLINE_DECLARES_
 	    [
 		#include "stdio.h"
 		#include "readline/readline.h"
+		#include "readline/history.h"
 	    ],
 	    [ 
 		HIST_ENTRY entry;
